app/models/member.rb

```diff
@@ -1,8 +1,21 @@
 class Member < ApplicationRecord
   AVERAGE_WEEKS_PER_MONTH = 4.33
   MONTHS_PER_YEAR = 12
+  PAYMENT_INTERVALS = [
+    "Hourly",
+    "Weekly",
+    "Every Two Weeks",
+    "Twice a Month",
```

Comment:
isn't it kind of repetitive and confusing to have twice a month and every 2 weeks as options?

Reply:
It is. I don't know who's in charge of coming up with these options but I do know they crazy

Reply:
do we want to include this data in our model, then? or should we just ignore than option for MI Bridges?

Reply:
ah I see you removed "Yearly" as an option. I think we want to keep that for our integration with the PDF, which has a checkbox for "Yearly" when you report salary.

If we keep that, then we can flip the conditional and assume that Yearly payment means salary and anything else is hourly, and we can report the payment interval and hours per week for anyone not on a salary.

This validation is awesome, but I do think we should remove the options we don't want to make available in our app, right?

Reply:
> do we want to include this data in our model, then? or should we just ignore that option for MI Bridges? It would be nice to not allow this kind of insane data in our DB

I mean ... there's a big overlap when you consider the two - "every 2 weeks" versus "twice a month", but I've been at jobs where it's been one and not the other. And then at another job where it's the opposite. It's dumb, but I think it's valid.

I do remember more than a handful of times where I'd get paid 3 times in a month bc of the every 2 weeks thing and think "YEAH!"

And then at the twice a month jobs where sometimes those long ass 31-day months would seriously harsh my mellow.

Reply:
ok that's fine -- I guess I'd maybe check with someone on the design side

made a card for that here so we can punt for now https://trello.com/c/PLcgU178